### DISCUSSION TITLE
res.render support?

### DISCUSSION THREAD

**Discussion by harshmandan** [nanoexpress/nanoexpress#238] [Ideas] [answered]
### Is you/your team sponsoring this project

- [ ] Yes
- [x] No

## Request

Hello, is there any way I can render EJS/PugJS templates like `res.render()` function of expressJS. Maybe even as an optional plugin?

Thanks!

**Comment by dalisoft** [comment]
Hi @harshmandan 
Thanks for interesting. Happy new year!

Sorry, but there no support currently and not planned yet, but you can create your middleware to provide support on middlewares repository and I’ll check and merge if it meets requirements. 
But implementing new one I don’t have a lot of times, so I don’t know how much time it takes. 
On my next plan fastest file upload middleware, GraphQL improvements and then maybe handlebars support

**Comment by harshmandan** [comment]
Okay!
Thanks for the reply. I also think that file-upload support is more important for now (Waiting for it). Also, maybe add `multipart/form-data` support as well.
Thanks!

**Comment by dalisoft** [comment]
Thank you for interesting to library. 
Would be nice to keep issue opened as in future this feature may be implemented

**Comment by dalisoft** [comment, marked as answer]
File upload is implemented, but there more than just simple tasks to implement this feature. If anyone interested, you can create middleware, i'll add to [nanoexpress/middlewares](https://github.com/nanoexpress/middlewares) repository
